### PR TITLE
stm32-f4-discovery: Fix acm not working in usbhost mode

### PR DESCRIPTION
**Pull request by satheesh-ks**
The patch will fix the listed issues

1) Cant enable usbhost CDCACM mode from menuconfig
2) Cant enable any of the USB logs from menuconfig
3) with "stm32-f4-discovery" board, /dev/ttyACM0 node not populating creating in usb host mode.
     3.1) Verified the Tx case from stm32 board to connected device, thru LeCroy usb pkt analyser.
     3.2) Cant verify the Rx case, as no suitable test device not found.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [f84cd4e622c110fa955a41df11a77b2c968541f2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [f84cd4e622c110fa955a41df11a77b2c968541f2] - Code Rule Check OK.

File: os/Kconfig.debug [hunk 1]
```diff
@@ -785,6 +785,38 @@ config DEBUG_TASK_MAX_COUNT
 
 endif #DEBUG_SYSTEM
 
+config DEBUG_USB
```
Comment (sunghan-chang) [inline os/Kconfig.debug]: Please locate this after `DEBUG_TTRACE`. There is a section of driver.

**Comment by sunghan-chang** [comment]
@satheesh-ks @sijupunnnoose Could you check my comment? After relocate, I will merge.

**Comment by seinfra** [comment]
Target : [9cd670a49a5f757aa9bbed0d86320868f30412d2] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9cd670a49a5f757aa9bbed0d86320868f30412d2] - Code Rule Check OK.

File: os/Kconfig.debug (hunk 1 above)
Comment (satheesh-ks) [reply os/Kconfig.debug]: @sunghan-chang: corrected by moving below DEBUG_TTRACE

**Pull request merged**